custom_components/powercalc/data/signify/5060730P7_01/model.json

```diff
@@ -0,0 +1,14 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly 1PM",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "master"
+    },
+    "name": "Hue Centris 4-spot ceiling",
+    "standby_power": 0.4,
+    "aliases": []
```

Comment:
Also known on following zigbee id's:
Zigbee ID: 5060730P7_01 | 5060730P7_02 | 5060730P7_03 | 5060730P7_04 | 5060730P7_05
See https://zigbee.blakadder.com/Philips_5060730P7.html
Set directory name to 5060730P7